### PR TITLE
feat(utils): add hdfs_user_homes role and playbook

### LINKED ISSUE #437: Add utils role and playbook to create HDFS users home directory
Create a role for creating HDFS users home directory. Create a playbook that use this role.

### PR DESCRIPTION
**Pull request by rpignolet**
#### Which issue(s) this PR fixes:

Fixes #437 

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Comment by rpignolet** [comment]
I'm not sure about the variable name `hdfs_users`, maybe `hdfs_users_home` is better because the role create the HDFS home directory only.

**Comment by gboutry** [comment]
> I'm not sure about the variable name `hdfs_users`, maybe `hdfs_users_home` is better because the role create the HDFS home directory only.

I agree, it doesn't make sense, renaming it `hdfs_homes` or something like that would make more sense

**Comment by rpignolet** [comment]
Rename `hdfs_users` to `hdfs_user_homes`. I prefer to keep `user` in the variable name in case we have service homes or other.

**Review by gboutry — APPROVED** [review]
> Rename hdfs_users to hdfs_user_homes. I prefer to keep user in the variable name in case we have service homes or other.

I agree, it's fine by me

File: roles/utils/hdfs_user_homes/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+hdfs_user_homes: []
+  # - user: tdp_user
+  #   group: tdp_user
+  #   mode: "700" # default
+  #   path: "/user/tdp_user" # default
```
Comment (Nuttymoon) [inline roles/utils/hdfs_user_homes/defaults/main.yml:9]:
```suggestion
  #   path: "/user/{{ user }}" # default
```

Comment (rpignolet) [reply roles/utils/hdfs_user_homes/defaults/main.yml]: Done

**Review by Nuttymoon — APPROVED** [review]

**Pull request merged**